chat-client/package.json

```diff
@@ -22,8 +22,8 @@
     },
     "dependencies": {
         "@aws/chat-client-ui-types": "^0.1.28",
-        "@aws/language-server-runtimes-types": "^0.1.23",
-        "@aws/mynah-ui": "^4.31.0-beta.6"
+        "@aws/language-server-runtimes-types": "^0.1.25",
+        "@aws/mynah-ui": "^4.31.0-beta.7"
```

Comment:
I had to update it, because 6 did not contain `fileDetails` yet. It does not impact the final bundle as we always include latest Mynah there.